qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -215,6 +216,21 @@ def load_account(self) -> Optional[AccountProvider]:
 
         return default_provider
 
+    def load_providers(self,
```

Comment:
This new lazy loading should be transparent to users - i.e. they shouldn't have to do a separate `_load_providers()` call.

Reply:
Fixed!